src/server/storage.ts

```diff
@@ -129,6 +130,11 @@ export class GameStorage {
    */
   public mobList: Map<MobId, Entity> = new Map();
 
+  /**
+   * Logged-in user entities.
+   */
+  public userList: Map<UserId, Entity> = new Map();
```

Comment:
There is a potential problem with the rapid, limitless growth of this storage. An ill-wisher can quickly send Login requests with random session ids, which will cause memory to be filled with empty profiles of fake users. Since we don't have a mechanism of session validation, we can complicate the task for a potential intruder. After the player is disconnected, we can delete those profiles that have 0 kills. There is a small risk that we will lose the data about deaths if a newcomer at the first game can not kill anyone, but I do not think it is significantly (or we can add a condition for the time of the active player's game, for example, as it is done for votemute restriction). So, to take advantage of the vulnerability, the ill-wisher will have to spend resources (at least time), not just connect a still bot and wait for someone to kill it.

Reply:
For the "/whois" idea, it is already desirable to have an external database. Would it be useful here too? Something like LMDB or LevelDB, but maybe SQLite is also fine if the node.js bindings are implemented well

Reply:
If we decide to do something like a single sessions server as I described above, we will definitely need the database. Congratulatio already uses SQLite in [login server](https://github.com/airmash-refugees/airmash-backend/blob/master/servers/login.js), so I wrote that we might take it as a basis and modify it.

Reply:
Agreed, I am thinking in a future change, this Map should be used to store only the User entities referenced by currently connected or awaiting potential recovery Player entities, with all other stats stored remotely and/or locally cached to disk.